### ISSUE TITLE
New Haven Network dataset is mis-named. 

### ISSUE THREAD

**Issue by ljwolf** [pysal/libpysal#360]
the files included in the `newHaven` data are mis-named. The `newhaven_neworks.*` file should become `newhaven_network.*`. Not sure if that breaks example code elsewhere.

**Comment by jGaboardi** [comment]
Since the `newHaven` dataset is a `remote` that resides in @sjsrey's [`newHaven`](https://github.com/sjsrey/newHaven) repo that would have to be renamed, correct?

**Comment by sjsrey** [comment]
> Since the `newHaven` dataset is a `remote` that resides in @sjsrey's [`newHaven`](https://github.com/sjsrey/newHaven) repo that would have to be renamed, correct?

Yes, it is remote and the change should be made there. That error has been there since PySAL 1.13 at least, so I'm imagining some breakage when the name is corrected.

**Comment by jGaboardi** [comment]
xref #466
